pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -476,44 +473,77 @@ func (c *Controller) addNodeRoute(nodeName string, node *corev1.Node) error {
 			return nil
 		}
 		peerGatewayIP := ip.NextIP(peerPodCIDRAddr)
-		peerConfig[peerPodCIDR] = peerGatewayIP
-		podCIDRs = append(podCIDRs, peerPodCIDR)
+		peerConfigs[peerPodCIDR] = peerGatewayIP
+		peerPodCIDRs = append(peerPodCIDRs, peerPodCIDR)
+		peerNodeIP := peerNodeIPs.IPv4
+		family := "IPv4"
+		if peerGatewayIP.To4() == nil {
+			peerNodeIP = peerNodeIPs.IPv6
+			family = "IPv6"
+		}
+
+		klog.InfoS("Adding route and flow to Node", "Node", nodeName, "podCIDR", podCIDR,
+			"peerNodeIP", peerNodeIP, "IPFamily", family)
```

Comment:
nit: IPFamily doesn't provide more information as there are already podCIDR and peerNodeIP.

Reply:
removed